specs/angular-readonly-form-styling/README.md

```diff
@@ -0,0 +1,118 @@
+# Angular read-only form styling
```

Comment:
Is anything in the proposal Angular-specific anymore? If not, could you update the document and PR titles to be more generic?